components/tab/src/tab-bar/scroll-bar.js

```diff
@@ -204,6 +225,7 @@ class ScrollBar extends Component {
 ScrollBar.propTypes = {
     children: propTypes.node.isRequired,
     dataTest: propTypes.string.isRequired,
+    autoHideScollButtons: propTypes.bool,
```

Comment:
There's a typo here: `autoHideScollButtons` => `autoHideScrollButtons`
This change would have to be made in various places, if we should chose to keep this prop.

Reply:
Removed